core/src/main/scala/org/apache/spark/util/ManualClock.scala

```diff
@@ -17,10 +17,33 @@
 
 package org.apache.spark.util
 
-class FakeClock extends Clock {
-  private var time = 0L
+class ManualClock(private var time: Long) extends Clock {
```

Comment:
This could probably be `private[spark]`. Also could you add some javadocs for these classes?